app/src/scenes/inscription/Create/stepProfil.js

```diff
@@ -273,7 +273,7 @@ export default () => {
                           <Field
                             as="select"
                             validate={(v) => !v && requiredMessage}
-                            disabled={values.birthCityZip.length !== 5}
+                            disabled={values.birthCityZip?.length !== 5}
```

Comment:
En fait là ça va pas être disabled si birthCityZip est pas set. Alors qu'en vrai ça DOIT être disabled

```suggestion
                            disabled={(values.birthCityZip || []).length !== 5}
```